src/modules/files-storage/domain/mapper/wopi.builder.ts

```diff
@@ -0,0 +1,49 @@
+import { EntityId } from '@shared/domain/types';
+import { AccessUrl } from '../access-url.vo';
+import { WopiAccessToken } from '../wopi-access-token.vo';
+import { WopiPayload } from '../wopi-payload.vo';
+
+export class WopiBuilder {
```

Comment:
Die Klasse ist kein Builder. 
Sonst müsste irgendwo das build(): ZIEL_TYP auftauchen und alle anderen Methoden wären XYZ(): this.

Es werden auch 3 verschiedenen Sachen durch public returned.
Also entweder muss das zerlegt werden:
- WopiAccessToken Factory
- WopiPayloadFactory
- AccessUrlFactory
...bei uns dann jeweils mit .build() aber eigentlich eher mit .create() als Methode, oder man macht es etwas unsauberer und nennt die Klasse Mapper.